packages/login/src/vaadin-login-form-mixin.js

```diff
@@ -16,10 +16,12 @@ export const LoginFormMixin = (superClass) =>
     }
 
     /** @protected */
-    connectedCallback() {
+    async connectedCallback() {
       super.connectedCallback();
 
       if (!this.noAutofocus) {
+        // Wait for the form to fully render.
+        await new Promise(requestAnimationFrame);
```

Comment:
From briefly testing this, it seems that using `await this.updateComplete` works here too.